lib/tasks/after_tasks.rake

```diff
@@ -874,6 +874,42 @@ namespace :After do
     puts("Added default rating to works: #{updated_works}") && STDOUT.flush
   end
 
+  desc "Delete invalid icon data from pseuds"
+  task(delete_invalid_pseud_icon_data: :environment) do
+    # From validates_attachment_content_type in pseuds model.
+    valid_types = %w[image/gif image/jpeg image/png]
+
+    # If you change either of these, update lookup_invalid_pseuds.rb in
+    # otwcode/otw-scripts to ensure the proper users are notified.
+    pseuds_with_invalid_icons = Pseud.where("icon_file_name IS NOT NULL AND icon_content_type NOT IN (?)", valid_types)
+    pseuds_with_invalid_text = Pseud.where("CHAR_LENGTH(icon_alt_text) > ? OR CHAR_LENGTH(icon_comment_text) > ?", ArchiveConfig.ICON_ALT_MAX, ArchiveConfig.ICON_COMMENT_MAX)
+
+    invalid_pseuds = [pseuds_with_invalid_icons, pseuds_with_invalid_text].flatten.uniq
+    invalid_pseuds_count = invalid_pseuds.count
+
+    # Update the pseuds.
+    puts("Updating #{invalid_pseuds_count} pseuds") && STDOUT.flush
+
+    invalid_pseuds.each do |pseud|
+      # Delete the icon if it's not a valid type.
+      unless valid_types.include?(pseud.icon_content_type)
+        pseud.icon = nil
+      end
+
+      # Delete the icon alt text if it's too long.
+      if pseud.icon_alt_text.length > ArchiveConfig.ICON_ALT_MAX
+        pseud.icon_alt_text = ""
+      end
+
+      # Delete the icon comment if it's too long.
+      if pseud.icon_comment_text.length > ArchiveConfig.ICON_COMMENT_MAX
```

Comment:
Hound ignores line length because we told it to: https://github.com/otwcode/otwarchive/blob/678bf364c791a503d4c9647cf9bed67daace3bea/.rubocop.yml#L24-L25